hack/api-reference/template/members.tpl

```diff
@@ -0,0 +1,48 @@
+{{ define "members" }}
+
+{{ range .Members }}
+{{ if not (hiddenMember .)}}
+<tr>
+    <td>
+        <code>{{ fieldName . }}</code></br>
+        <em>
+            {{ if linkForType .Type }}
+                <a href="{{ linkForType .Type}}">
+                    {{ typeDisplayName .Type }}
+                </a>
+            {{ else }}
+                {{ typeDisplayName .Type }}
+            {{ end }}
+        </em>
+    </td>
+    <td>
+        {{ if fieldEmbedded . }}
+            <p>
+                (Members of <code>{{ fieldName . }}</code> are embedded into this type.)
+            </p>
+        {{ end}}
+
+        {{ if isOptionalMember .}}
+            <em>(Optional)</em>
+        {{ end }}
+
+        {{ safe (renderComments .CommentLines) }}
+
+    {{ if and (eq (.Type.Name.Name) "ObjectMeta") }}
+        Refer to the Kubernetes API documentation for the fields of the
```

Comment:
this text looks like something that is K8s API doc related.. im not sure how and where this is used though

Reply:
sorry for the delay, what's your question?

I didn't write this file, I copied it from https://github.com/ahmetb/gen-crd-api-reference-docs as it seems to be how it's meant to be used. My understanding is that this specific part generates a generic description for the `ObjectMeta` fields in API resources. See https://antrea.io/docs/master/api-reference/.